packages/ui/src/hooks/use-social-connector.ts

```diff
@@ -0,0 +1,86 @@
+import { useEffect, useCallback } from 'react';
+import { useLocation } from 'react-router-dom';
+
+import { signInWithSocial, signInToSoical } from '@/apis/social';
+import { generateRandomString } from '@/utils';
+
+import useApi from './use-api';
+
+const StorageKeyPrefix = 'social_auth_state';
+const WebPlatformPrefix = 'web';
+const MobilePlatformPrefix = 'mobile';
```

Comment:
Should use `camelCase` instead of `PascalCase` for consts, right?

Reply:
yes updated